### ISSUE TITLE
Cannot Restore Package

### ISSUE THREAD

**Issue by krasninja** [Youssef-ben/inbucket-csharp-client#6]
Cannot restore the package to the project.

```
PM> install-package Inbucket.CSharp.Client
Restoring packages for D:\work\...\Project.csproj...
  GET https://api.nuget.org/v3-flatcontainer/inbucket.csharp.client/index.json
  OK https://api.nuget.org/v3-flatcontainer/inbucket.csharp.client/index.json 523ms
  GET https://api.nuget.org/v3-flatcontainer/inbucket.csharp.client/1.1.0/inbucket.csharp.client.1.1.0.nupkg
  OK https://api.nuget.org/v3-flatcontainer/inbucket.csharp.client/1.1.0/inbucket.csharp.client.1.1.0.nupkg 81ms
Installing Inbucket.CSharp.Client 1.1.0.
install-package : Cannot create "C:\Users\ivan\.nuget\packages\inbucket.csharp.client\1.1.0\LICENSE.txt" because a file or directory with the same name already exists.
At line:1 char:1
+ install-package Inbucket.CSharp.Client
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Time Elapsed: 00:00:01.4986826
```

**Comment by Youssef-ben** [comment]
Hi @krasninja, Thank you for reporting this problem, will try to fix it as soon as possible.

**Comment by Youssef-ben** [comment]
Hi @krasninja, the problem should be fixed with the new release [v1.1.1](https://www.nuget.org/packages/Inbucket.CSharp.Client/). 

Thank you for your patience, Closing the issue.

**Comment by krasninja** [comment]
@Youssef-ben Thank you!
